src/Azure.Functions.Cli/Helpers/DotnetHelpers.cs

```diff
@@ -33,12 +33,12 @@ public static void EnsureDotnet()
         /// <param name="projectDirectory">Directory containing the .csproj file</param>
         /// <returns>Target framework, e.g. net8.0</returns>
         /// <exception cref="CliException"></exception>
-        public static async Task<string> DetermineTargetFramework(string projectDirectory)
+        public static async Task<string> DetermineTargetFramework(string projectDirectory, string projectFilename = "")
```

Comment:
set the default value of parameter to `null`. Update the method summary to include the new parameter added.

`, string projectFileName = null`